efficacy/app.R

```diff
@@ -12,6 +11,8 @@ data <- within(data, {
   library(scda)
   library(scda.2022)
   library(nestcolor)
+  # optional libraries
+  library(sparkline)
```

Comment:
apologies that I jumped into the already closed PR
may I ask: why this particular change?
AFAIK sparkline is a supplementary *module* package - it's not required for data loading part (as opposed to `dplyr`, `scda` etc. which are correct here in this context). Therefore I would expect it to be at the beggining of the app.R script and not inside `within()`.

(same for safety/app.R)

Reply:
I thought that loading this library would improve the variable browser module.

Reply:
That's right. It's needed for the variable browser. I got the same understanding. But the position is quite unusual now because you moved the library into the `teal_data(...) %>% within(<here>)` and usually we have modules libs load outside of this at the top of the script.